test/server/utils/new-line-replacer.js

```diff
@@ -0,0 +1,3 @@
+exports.replace = function (content) {
```

Comment:
Looking at this now, it shouldn't be even named `replaceNewLines` it's too generic for this function, because:
 - it works with buffers only
 - it replaces with '\r\n`
Both are specific for formdata, so it should be called `normalizeFormDataNewLines`